test/hotspot/jtreg/gc/stress/systemgc/TestSystemGC.java

```diff
@@ -191,9 +191,9 @@ public static void main(String[] args) throws Exception {
         if (args.length == 0) {
             throw new IllegalArgumentException("Must specify timeout in seconds as first argument");
         }
-        int timeout = Integer.parseInt(args[0]) * 1000;
-        System.out.println("Running with timeout of " + timeout + "ms");
-        endTime = System.currentTimeMillis() + timeout;
+        long timeoutNanos = Integer.parseInt(args[0]) * 1_000_000_000L;
+        System.out.println("Running with timeout of " + timeoutNanos + "ns");
```

Comment:
The precision here is in seconds. Maybe just leave this print out to be in ms?

Reply:
I will change it to seconds.